src/Axiom.ts

```diff
@@ -2,8 +2,8 @@ import { ProgramManager } from "./ProgramManager";
 
 declare const SGWorld: ISGWorld;
 export const sgWorld = SGWorld;
-export const basePath = "\\\\192.168.1.19/d/C-ARMSAS/axiom/";
-// export const basePath = "C:/dev/Github/AxiomSkylineHolotable/dist/Axiom/";
+// export const basePath = "\\\\192.168.1.19/d/C-ARMSAS/axiom/";
+ export const basePath = "C:/dev/Github/AxiomSkylineHolotable/dist/Axiom/";
```

Comment:
Might want to change this back

Reply:
@rwillsonAxiom done. I can't think of a decent way to allow devs who aren't on site to store that. Since we aren't in a server environment we can't load files (like json etc) to store config. 

Only option I can thing is to add a task runner in like Grunt to manage the build task. But didn't really want to add the complication at this stage.

If you have any better ideas let me know, until then I will try my best to remember